src/app/client/src/app/plugins/profile/components/update-user-details/update-user-details.component.ts

```diff
@@ -22,10 +22,17 @@ export class UpdateUserDetailsComponent implements OnInit, OnDestroy {
   sbFormBuilder: FormBuilder;
   enableSubmitBtn = false;
   showDistrictDivLoader = false;
-  submitInteractEdata: IInteractEventEdata;
+  submitNameInteractEdata: IInteractEventEdata;
+  submitStateInteractEdata: IInteractEventEdata;
   telemetryInteractObject: IInteractEventObject;
   selectedState;
   selectedDistrict;
+  stateControl: any;
```

Comment:
I have told you to use a single variable for all the below variables.
example:
forChanges = [{key: state, oldValue: "", newValue: ""}, {key: district, oldValue: "", newValue: ""}]